.github/workflows/gosec.yaml

```diff
@@ -1,25 +0,0 @@
-name: Gosec
```

Comment:
I think that perhaps this file should have more comments explaining the purpose behind this GH action.

The idea was to have a cutting-edge go-sec check that runs against go-sec master branch while allowing to (at times ) be unstable. So even if it fails it should not meant a stop-of-the-world for the CD/CI.

Perhaps keeping this pipeline feature isolated from other stabler linting features could be a good idea.